src/lib/widget/index.svelte

```diff
@@ -1,7 +1,7 @@
 <script context="module" lang="ts">
   import { widgetApiFactory } from './_utilities/api.utilities';
   import { componentApi } from './_utilities/component.utilities';
-
+  export { PIProtect } from '@forgerock/ping-protect';
```

Comment:
Was this intentional?

Reply:
Oh, okay, I see. I may have been unclear about my position on this. I didn't mean to pass the whole class as is, but just the methods, to keep it consistent with the other Widget APIs. 

In other words, in the API utility module, you'd do [something similar to what we did for the `request` method](https://github.com/ForgeRock/forgerock-web-login-framework/blob/beta/src/lib/widget/_utilities/api.utilities.ts#L339): 

```js
  return {
    component: componentApi,
    configuration,
    getStores,
    journey,
    protect: {
      start: PIProtect.start.bind(PIProtect),
      resume: PIProtect.resumeBehavorialData.bind(PIProtect),
      pause: PIProtect.pauseBehavorialData.bind(PIProtect),
    },
    request: HttpClient.request.bind(HttpClient),
    user,
  };
```

That way, we treat all APIs, even ones that are just passthroughs, consistently for our devs.